lisa/transformers/kernel_source_installer.py

```diff
@@ -51,6 +50,12 @@ class RepoLocationSchema(LocalLocationSchema):
     # fail the run if code exists
     fail_on_code_exists: bool = False
     cleanup_code: bool = False
+    auth_token: Optional[str] = field(
```

Comment:
It's ok to add it. Just FYI, internal repo can be used directly, if the running account is granted. It doesn't need auth token.